qa/L0_stability_metrics/check_results.py

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) 2021 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import sys
+import yaml
+import os
+import glob
+
+
+class TestOutputValidator:
+    """
+    Functions that validate the output
+    of the test
+    """
+
+    def __init__(self, config, test_name, results_path, tolerance):
+        self._config = config
+        self._models = list(config['profile_models'])
+        self._result_path = results_path
+        self._tolerance = tolerance
+
+        check_function = self.__getattribute__(f'check_{test_name}')
+
+        if check_function():
+            sys.exit(0)
+        else:
+            sys.exit(1)
+
+    def check_metrics_stability(self):
+        """
+        Makes sure that the same configuration
+        appears as the best across iterations for
+        each model
+        """
+
+        # There should be 4 csv files the results path
+        pathname = os.path.join(self._result_path, 'result_*.csv')
+        csv_contents = []
```

Comment:
Maybe add a check to make sure that 4 CSV files are generated?

Reply:
Sorry that comment is out of place. I will remove it. The number of csv files is set by the test.sh